pkg/zonemgr/zone_file_cache.go

```diff
@@ -55,10 +56,33 @@ func (zoneFileCache *ZoneFileCache) prepare() {
 	zoneFileCache.generateHeaderSuffix()
 	zoneFileCache.soaSerial = 0
```

Comment:
please read the file here, and set the serial either 0 or whatever the file has if exists
then no need this part
https://github.com/kubevirt/kubesecondarydns/pull/45/files#diff-91aeb36dc70c24eda218702f6964e34e7cbb3c7f29cce4f45ed3b09b6f59cb11R44

Reply:
It is fine that there will be an helper function that reads the file in order to determine the SOA to start from.
It is better imo than to create the header twice

Reply:
done